receipt_upload/receipt_upload/label_sync/union.py

```
@@ -0,0 +1,276 @@
+                source_proposed_by=getattr(sl, "label_proposed_by", "") or "",
+            )
+        )
+    return adds
+
+
+def apply_union(
+    adds: List[LabelAdd],
+    dynamo,
+    *,
+    apply: bool = False,
+    backup_path: str = None,
```

Ruff (0.15.18)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)